tests/ui_suite/fav_test.py

```diff
@@ -23,22 +32,33 @@ def test_add2fav(logined_page, fav_numbers: tuple, add_fav_from_detail: str):
                 fav_goods_names_expected.append(
                     LoginedPage.good_name_text(goods_for_fav[goods_index]))
                 # клик по сердцу - добавление в избранное
-                LoginedPage.goods_heart_button(
-                    goods_for_fav[goods_index]).click()
+                # with allure.step('Add desired goods to favorites using heart button'):
+                #     LoginedPage.goods_heart_button(goods_for_fav[goods_index]).click()
+                #     do_allure_attach(f'After clicking on the {goods_index}-th goods/s heart button')
```

Comment:
please delete comments those are not needed anymore

Reply:
done